cumulusci/cli/logger.py

```diff
@@ -1,9 +1,10 @@
 """ CLI logger """
 import logging
+import logging.handlers
```

Comment:
±: Did you mean?:
```suggestion
from logging.handlers import RotatingFileHandler
```
(with a corresponding change on line 55 of this file)

Otherwise I don't think you need this line.